tools/simulator/metrics-collector/README.md

```diff
@@ -17,37 +18,50 @@ You must meet the following requirements to setup metrics collector:
 	metrics-collector-deployment-74cbf5896f-jhg6v      1/1     Running   0          111m
 	```
 
-## Quick Start
+## How to use
+
 ### Setup metrics collector
-You can run `setup-metrics-collector.sh` following with a number to setup multiple metrics collector.
 
-For example, setup 2 metrics collectors with 100 workers by the following command:
-```
-# ./setup-metrics-collector.sh 2 100
-```
-Check if all the metrics collector running successfully in your cluster:
+1. Run `setup-metrics-collector.sh` script to setup multiple metrics collector, `-n` specifies the simulated metrics collector number, optional `-t` specifies the metrics data source type, can be "SNO"(default value) or "NON_SNO", and optional `-w` specifies the worker number for each simulated metrics collector, you can also specifies the simulated metrics collector name prefix by the `-m` flag. For example, setup 2 metrics collectors with 100 workers that collect the SNO metrics data by the following command:
+
+```bash
+# ./setup-metrics-collector.sh -n 2 -t SNO -w 100
 ```
-# oc get pods --all-namespaces | grep simulate-managed-cluster
+
+2. Check if all the metrics collector running successfully in your cluster:
+
+```bash
+# oc get pods --all-namespaces | grep simulated-managed-cluster
 simulate-managed-cluster1                          metrics-collector-deployment-7d69d9f897-xn8vz                    1/1     Running            0          22h
 simulate-managed-cluster2                          metrics-collector-deployment-67844bfc59-lwchn                    1/1     Running            0          22h
 ```
-It simulates 200 metrics collectors to push the data into hub thanos.
 
-> Note: if you want the simulated metrics-collector be scheduled to master node, so that more simulated metrics-collectors can be deployed, you can set the environment variable `ALLOW_SCHEDULED_TO_MASTER` to be `true` before executing the setup script.
+> _Note:_ the above command will simulate 200 metrics collectors pushing the data concurrently into hub thanos.
+
+> _Note:_ if you want the simulated metrics-collector be scheduled to master node, so that more simulated metrics-collectors can be deployed in one cluster, you can set the environment variable `ALLOW_SCHEDULED_TO_MASTER` to be `true` before executing the setup script.
```

Comment:
I think we do not need to have this Note since one metrics-collector pod can simulate multiple workers. for example, there are only 2 pods which can simulate 200 collectors.

Reply:
removed this note and tolerations section.